### PR TITLE
[FEAT] [BWS]: Clean fiat_rates2 periodically

### PR DESCRIPTION
**Pull request by rastajpa**

### CODE REVIEW

File: packages/bitcore-wallet-service/src/lib/cleanfiatrates.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+          console.log(`\tFiltering times for ${key.toUpperCase()}`);
+          let prevTime = null;
+          let isSameDay, isSameHour;
+
+          _.forEach(tsGroup, ts => {
+            if (prevTime) {
+              isSameDay = moment(prevTime).isSame(moment(ts), 'day');
+              isSameHour = moment(prevTime).isSame(moment(ts), 'hour');
+
+              if (!isSameDay || (isSameDay && !isSameHour)) {
```
Comment (matiu) [inline packages/bitcore-wallet-service/src/lib/cleanfiatrates.ts:120]: if `results` are order in time `!isSameHour` should be enough, right?

File: packages/bitcore-wallet-service/src/lib/cleanfiatrates.ts [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+        // keep one date every hour for each coin
+        _.forEach(tsGruopedByCoin, (tsGroup, key) => {
+          console.log(`\tFiltering times for ${key.toUpperCase()}`);
+          let prevTime = null;
+          let isSameDay, isSameHour;
+
+          _.forEach(tsGroup, ts => {
+            if (prevTime) {
+              isSameDay = moment(prevTime).isSame(moment(ts), 'day');
```
Comment (matiu) [inline packages/bitcore-wallet-service/src/lib/cleanfiatrates.ts:117]: maybe we should add `if (prevTime > ts) =>` ERROR, results not in order`

Comment (matiu) [reply packages/bitcore-wallet-service/src/lib/cleanfiatrates.ts:117]: or if (prevTime > ts + 60*1000) (give it 60 secs JIC) => error

File: packages/bitcore-wallet-service/src/lib/cleanfiatrates.ts (hunk 1 above)
Comment (rastajpa) [reply packages/bitcore-wallet-service/src/lib/cleanfiatrates.ts:120]: yes! you're right!

File: packages/bitcore-wallet-service/src/lib/cleanfiatrates.ts (hunk 2 above)
Comment (rastajpa) [reply packages/bitcore-wallet-service/src/lib/cleanfiatrates.ts:117]: sounds good to me!

**Pull request merged**